GETTING_STARTED.md

```diff
@@ -0,0 +1,875 @@
+# Getting Started
+
+## 🆘 Macrometa Support
+
+If you have any trouble or need help while using SDK please contact
+[support@macrometa.com](mailto:support@macrometa.com).
+
+---
+
+## 📃 Macrometa Document Explained
+
+One of the datastore's that Macrometa supports is **document**, a **document**
+is an object that is a JSON serializable object with the following properties:
+
+- Contains the `_key` field, which identifies the document uniquely within a
+  specific collection.
+- Contains the `_id` field (also called the _handle_), which identifies the
+  document uniquely across all collections within a fabric. This ID is a
+  combination of the collection name and the document key using the format
+  `{collection}/{key}` (see example below).
+- Contains the `_rev` field. GDN supports MVCC (Multiple Version Concurrency
+  Control) and is capable of storing each document in multiple revisions. Latest
+  revision of a document is indicated by this field. The field is populated by
+  GDN and is not required as input unless you want to validate a document
+  against its current revision.
+
+Here is an example of a valid document:
+
+```
+    {
+        '_id': 'students/bruce',
+        '_key': 'bruce',
+        '_rev': '_Wm3dzEi--_',
+        'first_name': 'Bruce',
+        'last_name': 'Wayne',
+        'address': {
+            'street' : '1007 Mountain Dr.',
+            'city': 'Gotham',
+            'state': 'NJ'
+        },
+        'is_rich': True,
+        'friends': ['robin', 'gordon']
+    }
+```
+
+**Edge documents (edges)** are similar to standard documents but with two
+additional required fields `_from` and `_to`. Values of these fields must be the
+handles of "from" and "to" vertex documents linked by the edge document in
+question (see [Graph](docs/Reference/Graph/README.md) for details).
+
+Here is an example of a valid edge document:
+
+```
+    {
+        '_id': 'friends/001',
+        '_key': '001',
+        '_rev': '_Wm3dyle--_',
+        '_from': 'students/john',
+        '_to': 'students/jane',
+        'closeness': 9.5
+    }
+```
+
+---
+
+## 🔐 Auth - Connect to GDN
+
+The first step to start using GDN is establishing a connection to
+[play.macrometa.io](https://play.macrometa.io/). When this code executes, it
```

Comment:
```suggestion
[play.paas.macrometa.io](https://play.paas.macrometa.io/). When this code executes, it
```

Reply:
Both play.paas and play work but since we have mentioned play.paas in website docs we can stick with that maybe